data/json/itemgroups/packages.json

```
@@ -0,0 +1,2567 @@
+      [ "package_large_vehicle_parts13", 35 ],
+      [ "package_large_vehicle_parts14", 15 ],
+      [ "package_large_vehicle_parts15", 15 ],
+      [ "package_large_vehicle_parts16", 25 ],
+      [ "package_large_vehicle_parts17", 10 ],
+      [ "package_large_vehicle_parts18", 10 ],
+      [ "package_large_vehicle_parts19", 10 ],
+      [ "package_large_vehicle_parts20", 15 ],
+      [ "package_large_vehicle_parts21", 10 ],
+      [ "package_large_vehicle_parts22", 10 ],
+      [ "package_large_vehicle_parts23", 10 ],
+      [ "package_large_vehicle_parts24", 20 ],
+      [ "package_large_vehicle_parts25", 10 ],
+      [ "package_large_vehicle_parts26", 30 ]
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "packages_large_materials",
+    "items": [ [ "package_large_materials1", 20 ], [ "package_large_materials2", 20 ], [ "package_large_materials3", 10 ] ]
+  },
+  {
+    "type": "item_group",
+    "id": "packages_large_other",
+    "items": [
+      [ "package_large_other1", 10 ],
+      [ "package_large_other2", 10 ],
+      [ "package_large_other3", 20 ],
+      [ "package_large_other4", 20 ],
+      [ "package_large_other5", 15 ],
+      [ "package_large_other6", 15 ],
+      [ "package_large_other7", 10 ]
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "package_small_clothing_underwear1",
+    "subtype": "collection",
+    "items": [ { "item": "boxer_briefs", "count": 2 } ]
+  },
+  {
+    "type": "item_group",
+    "id": "package_small_clothing_underwear2",
+    "subtype": "collection",
+    "items": [ { "item": "boxer_shorts", "count": 2 } ]
+  },
+  {
+    "type": "item_group",
+    "id": "package_small_clothing_underwear3",
+    "subtype": "collection",
+    "items": [ { "item": "briefs", "count": 2 } ]
+  },
+  {
+    "type": "item_group",
+    "id": "package_small_clothing_underwear4",
+    "subtype": "collection",
+    "items": [ { "item": "bra", "count": 2 } ]
+  },
+  {
+    "type": "item_group",
+    "id": "package_small_clothing_underwear5",
+    "subtype": "collection",
+    "items": [ { "item": "sports_bra", "count": 2 } ]
+  },
+  {
+    "type": "item_group",
+    "id": "package_small_clothing_underwear6",
+    "subtype": "collection",
+    "items": [ { "item": "panties", "count": 2 } ]
+  },
+  {
+    "type": "item_group",
+    "id": "package_small_clothing_underwear7",
+    "subtype": "collection",
+    "items": [ { "item": "boy_shorts", "count": 2 } ]
+  },
+  {
+    "type": "item_group",
+    "id": "package_small_clothing_underwear8",
+    "subtype": "collection",
+    "items": [ { "item": "socks", "count": 3 } ]
+  },
+  {
+    "type": "item_group",
+    "id": "package_small_clothing_underwear9",
+    "subtype": "collection",
+    "items": [ [ "under_armor" ], [ "under_armor_shorts" ] ]
+  },
+  {
+    "type": "item_group",
+    "id": "package_small_clothing_underwear10",
+    "subtype": "collection",
+    "items": [ [ "long_undertop" ], [ "long_underpants" ] ]
```

There are numbers missing. Each entry of the "items" array should be an array with **two** elements: the item id and the probability.
```suggestion
    "items": [ [ "long_undertop", 100 ], [ "long_underpants", 100 ] ]
```
This applies to all the other "item_group" object here as well.